site/src/pages/ManagementSettingsPage/CustomRolesPage/CustomRolesPage.tsx

```diff
@@ -0,0 +1,71 @@
+import AddIcon from "@mui/icons-material/AddOutlined";
+import Button from "@mui/material/Button";
+import { type FC, useEffect } from "react";
+import { Helmet } from "react-helmet-async";
+import { useQuery } from "react-query";
+import { Link as RouterLink, useParams } from "react-router-dom";
+import { getErrorMessage } from "api/errors";
+import { organizationRoles } from "api/queries/roles";
+import { displayError } from "components/GlobalSnackbar/utils";
+import { PageHeader, PageHeaderTitle } from "components/PageHeader/PageHeader";
+import { useAuthenticated } from "contexts/auth/RequireAuth";
+import { useFeatureVisibility } from "modules/dashboard/useFeatureVisibility";
+import { pageTitle } from "utils/page";
+import CustomRolesPageView from "./CustomRolesPageView";
+
+export const CustomRolesPage: FC = () => {
+  const { permissions } = useAuthenticated();
+  const { assignOrgRole: canAssignOrgRole } = permissions;
+  const { custom_roles: isCustomRolesEnabled } = useFeatureVisibility();
+
+  const { organization = "default" } = useParams() as { organization: string };
```

Comment:
```suggestion
  const { organization } = useParams() as { organization: string };
```

This page is only accessible from routes with an `:organization` param.

Reply:
Steven mentioned today that custom roles should be independent from organizations and still accessible if organizations is not enabled